src/Core/src/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -101,6 +102,22 @@ public static IAppHostBuilder UseMauiServiceProviderFactory(this IAppHostBuilder
 			return builder;
 		}
 
+		public static IAppHostBuilder UseMicrosoftExtensionsServiceProviderFactory(this IAppHostBuilder builder)
```

Comment:
@rmarinho / @PureWeen - this one's for you. I think for now for Blazor scenarios having this utility method is critical. Once we figure out long term plans for DI we can revisit whether this is needed.